bot/scenarios/stomachPain.json

```diff
@@ -122,7 +130,7 @@
     {
       "id": "finalize",
       "type": "handler",
-      "data": { "name": "summary" }
+      "data": { "name": "summary.js" }
```

Comment:
To keep to you other changes, should be good to rename to:
"data": { "path": "summary.js" }